### PR TITLE
Slider branch

### PR DESCRIPTION
**Pull request by nayyyhaa**
Added slider component in inNotion-docs UI

### CODE REVIEW

File: docs/slider/index.html [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+              </div>
+                <div class="m-t-3">Value: <span class="output-slider-value"></span></div>
+              </code>
+            </pre>
+
+            <p class="m-v-2"><code>JS</code> snippet</p>
+            <pre class="code-container code-snippet m-v-1 p-1">
+              <code class="javascript fs-s">
+              /*---------SLIDER DEMO------------*/
+            const slider = document.querySelector(".slider");
```
Comment (MayankKumar10) [inline docs/slider/index.html:101]: remove commented code

**Review by MayankKumar10 — APPROVED** [review]
Just a suggestion everything else looks good to me.

**Review by vrinda-mahajan — APPROVED** [review]
Everything looks good to me. You can merge this PR.

**Comment by nayyyhaa** [comment]
Thanks @MayankKumar10 & @vrinda-mahajan for reviewing. Merging the PR.

**Pull request merged**